src/engraving/dom/measure.cpp

```diff
@@ -705,6 +706,23 @@ Segment* Measure::findFirstR(SegmentType st, const Fraction& t) const
     return 0;
 }
 
+Segment* Measure::getChordRestOrTimeTickSegment(const Fraction& f)
+{
+    Segment* seg = findSegment(SegmentType::ChordRest, f);
```

Comment:
Is it possible to call only getSegment and not call findSegment?

Reply:
Unfortunately no, because I need them to be searched in this specific order and do the next operation only if the previous one doesn't succeed